api/CcsSso.Core.Service/External/UserProfileService.cs

```diff
@@ -355,6 +359,20 @@ public async Task<UserEditResponseInfo> CreateUserAsync(UserProfileEditRequestIn
         IsRegisteredInIdam = isRegisteredInIdam
       };
     }
+    private async Task AddUserToDefaultGroup(int ciiOrgId, int UserId)
```

Comment:
private async Task AddUserToDefaultGroup(int orgId, int userId)

Reply:
Updated the method name and parameter to AddUserToDefaultGroup(int orgId, int userId)